napari/utils/settings/_defaults.py

```diff
@@ -413,6 +413,41 @@ class ApplicationSettings(BaseNapariSettings):
         description=trans._("Loop mode for playback."),
     )
 
+    grid_stride: int = Field(
+        default=1,
+        title=trans._("Grid Stride"),
+        description=trans._("Number of layers to place in each grid square."),
+        ge=-50,
+        le=50,
+        ne=0,
+    )
+
+    grid_width: int = Field(
```

Comment:
we should also be mirroring this on `GridCanvas.shape` (the field it's affecting).  That field is a `Tuple[int, int]` ... but these are two separate `int` fields.  I can see arguments for either... but it feels like we should make them consistent?  The validator should also apply there (which is another reason I liked the `conint(ne=0)` field)

Reply:
really, this is two ints because I started this whole thing building a widget with a separate QSpinBox for width and height.  Having the 2 ints in settings made it simple to get those QSpinBoxes in the preferences dialog, so I didn't think about it much.  If you want it to be a tuple in settings, I'll have to probably make more adjustments to get the preferences dialog to build properly.  I can do that, I think, if you prefer.